src/js/structs/Pod.js

```diff
@@ -138,6 +138,10 @@ module.exports = class Pod extends Service {
     return this._spec;
   }
 
+  getStatusString() {
+    return (this.get('status') || '').toUpperCase();
```

Comment:
We could lower-case if you prefer. I just think that upper-case strings seem more like a constant in the code and can be easily spotted.